### DISCUSSION TITLE
Is Etcd grpc proxy mature enough for production workload?

### DISCUSSION THREAD

**Discussion by joeyyang-db** [etcd-io/etcd#18094] [General]
Try luck here to find if there's any company that has adopted grpc proxy in production and stably run for a while. The grpc proxy exactly fits our use case but not sure if it's mature enough for production workload. Any input is appreciated!

**Comment by tjungblu** [comment]
Depends for what you want to use it. We're using it in OpenShift for a couple of years to proxy metrics to Prometheus, never seen an issue with that.
